app/src/app/components/contents/packets/PacketRunner.tsx

```diff
@@ -1,9 +1,31 @@
 import React from "react";
+import {useSelector} from "react-redux";
+import {RootState} from "../../../../types";
 
 export default function PacketRunner() {
+    const {packet, packets} = useSelector((state: RootState) => state.packets);
+    const data = Object.keys(packet).length > 0 ? packet : packets[0];
```

Comment:
This should handle the case where `packets` has length 0 as well. At the moment calls like `data.id` on line 13 will throw errors if `data` is null.